src/opm/input/eclipse/Schedule/KeywordHandlers.cpp

```diff
@@ -879,13 +879,13 @@ File {} line {}.)", wname, location.keyword, location.filename, location.lineno)
                   if (is_terminal_node) {
                       if (vfp_table > 0) {
                           std::string msg = fmt::format("The group {} is a terminal node of the network and should not have a vfp table assigned to it.", group_name);
-                          throw OpmInputError(msg, handlerContext.keyword.location());
+                          OpmLog::warning(OpmInputError::format(msg, handlerContext.keyword.location()));
```

Comment:
What is the reason that changes the `throw` to `warning`?  For fixed pressure nodes, we should not have a valid VFP table. I do not see this PR can change this.

Reply:
The reason is that we have existing datasets that rely on this being a warning only.. I can update the message to make it clear that the VFP will be ignored, but would very much prefer not throwing here.

Reply:
That will be fine. We can keep it as a warning.